### PR TITLE
New Crowdin translations

### PR DESCRIPTION
**Pull request by teolemon**
Enjoy!

### CODE REVIEW

File: fastlane/metadata/nl-BE/description.txt [hunk 1]
```diff
@@ -1,9 +1,9 @@
 
-▶ Find & Compare more than 930.000 products
+▶ Vind en vergelijk meer dan 930.000 producten
 
-The app allows to view the 930.000 products (including 175.000 in the US) already contained in the free and open database Open Food Facts, and also to add pictures and data for new products.
+De app maakt het mogelijk om de 930.000 producten (inclusief 175.000 uit de VS) in de gratis en open database Open Food Facts te bekijken, en om foto's en gegevens toe te voegen voor nieuwe producten.
 
-To discover the volunteer-run Open Food Facts project, visit https://world-en.openfoodfacts.org
+Om het vrijwillige Open Food Facts project te ontdekken, bezoek https://world-en.openfoodfacts.org
```
Comment (teolemon) [inline fastlane/metadata/nl-BE/description.txt]:
```suggestion
Om het vrijwillige Open Food Facts project te ontdekken, bezoek https://world-nl.openfoodfacts.org
```

File: fastlane/metadata/nl/description.txt [hunk 2]
```diff
@@ -73,7 +73,7 @@ Stel allergeenwaarschuwingen in
 
 ▶ Zeg hallo en/of word lid!
 
-Open Food Facts is also available on the web at https://world-en.openfoodfacts.org
+Open Food Facts is ook beschikbaar op het web op https://world-en.openfoodfacts.org
```
Comment (teolemon) [inline fastlane/metadata/nl/description.txt]:
```suggestion
Open Food Facts is ook beschikbaar op het web op https://world-nl.openfoodfacts.org
```

File: fastlane/metadata/ru/description.txt [hunk 3]
```diff
@@ -1,9 +1,9 @@
 
 ▶ Найдите и сравните более 930 000 продуктов
 
-The app allows to view the 930.000 products (including 175.000 in the US) already contained in the free and open database Open Food Facts, and also to add pictures and data for new products.
+Это приложение бесплатно дает доступ к базе данных Open Food Facts, в которой можно найти информацию о 930 тыс. продуктах (включая 175 тыс. в США), а также добавлять информацию и картинки новых продуктов.
 
-To discover the volunteer-run Open Food Facts project, visit https://world-en.openfoodfacts.org
+Чтобы узнать больше о проекте Open Food Facts, поддерживаемом на добровольных началах, посетите https://world-en.openfoodfacts.org
```
Comment (teolemon) [inline fastlane/metadata/ru/description.txt]:
```suggestion
Чтобы узнать больше о проекте Open Food Facts, поддерживаемом на добровольных началах, посетите https://world-ru.openfoodfacts.org
```

**Review by aleene — APPROVED** [review]
This seems ok to me

**Pull request merged**